gcs/project.py

```diff
@@ -182,6 +182,10 @@ def get_hmac_key(self, access_id):
             testbench.error.notfound("service account for key=%s" % access_id, None)
         return sa.get_key(key_id)
 
+    def list_hmac_keys(self):
+        """List keys across all service accounts in the project"""
+        return [sa.key_items() for sa in self.service_accounts.items() for sa in sa]
```

Comment:
Please add a test to make sure this is covered, it is a relative simple function, but let's try to keep the coverage high (it avoids hiding other errors).